tests/test_user.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+import requests
+from pymongo import MongoClient
+
+from query.user_query import GetUserResponse
+from query.user_query import UserQuery
+
+client = MongoClient("mongodb://localhost:27017/")
+test_db = client["connectify-testcases-db"]
+user_collection = test_db["user"]
+
+BASE_URL = "http://localhost:8000/graphql"
+
+
+@pytest.fixture(scope="module")
+def setup_database():
+    # Connect to the test database
+    # Insert test data
+    user_data = {"username": "testuser", "email": "test@example.com"}
+    test_db.user.insert_one(user_data)
+
+    # Provide the test database object
+    yield test_db
+
+    # Clean up after the tests by removing the inserted data
+    test_db.user.delete_one({"username": "testuser"})
```

Comment:
Todo: Updating the test block to store the data's used in testing (POST request) it in Mock Database.